packages/opentelemetry-resources/src/platform/node/detectors/AwsEc2Detector.ts

```diff
@@ -30,8 +30,18 @@ class AwsEc2Detector implements Detector {
    * See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-identity-documents.html
    * for documentation about the AWS instance identity document endpoint.
    */
+  readonly HTTP_HEADER = 'http://';
+  readonly AWS_IDMS_ENDPOINT = '169.254.169.254';
```

Comment:
Maybe `AWS_IDMS_HOST_ADDRESS`?

Reply:
Nice suggestion! 
But because we have another path called HOST_DOCUMENT. To avoid confusion, I believe this kind of naming is also acceptable

Reply:
It's IMDS (Instance Metadata Service) ;)

Reply:
Thank you for pointing out! Will modify this